README.md

```diff
@@ -6,112 +6,79 @@
 [![](https://img.shields.io/pypi/v/gokart)](https://pypi.org/project/gokart/)
 ![](https://img.shields.io/pypi/l/gokart)
 
-A wrapper of the data pipeline library "luigi".
+Gokart solves reproducibility, task dependencies, constraints of good code, and ease of use for Machine Learning Pipeline.
 
 
-## Getting Started
-Run `pip install gokart` to install the latest version from PyPI. [Documentation](https://gokart.readthedocs.io/en/latest/) for the latest release is hosted on readthedocs.
+[Documentation](https://gokart.readthedocs.io/en/latest/) for the latest release is hosted on readthedocs.
 
-## How to Use
-Please use gokart.TaskOnKart instead of luigi.Task to define your tasks.
 
+# Good thing about gokart
```

Comment:
Are there any non-goal / downside of the gokart?

Stating it keep users from misuse.

Reply:
@saiya Of course, there's no downside (kidding)

Reply:
added